packages/user/src/menu.ts

```diff
@@ -34,12 +34,21 @@ export class RendererUserMenu extends MenuBar.Renderer {
     let className = this.createItemClass(data);
     let dataset = this.createItemDataset(data);
     let aria = this.createItemARIA(data);
-    return h.li(
-      { className, dataset, tabindex: '0', onfocus: data.onfocus, ...aria },
-      this._createUserIcon(),
-      this.renderLabel(data),
-      this.renderIcon(data)
-    );
+
+    if ('empty' in dataset && dataset.empty === 'true') {
+      return h.li(
+        { className, dataset, tabindex: '0', onfocus: data.onfocus, ...aria },
+        this._createUserIcon(),
+        this.renderLabel(data)
+      );
+    } else {
+      return h.li(
+        { className, dataset, tabindex: '0', onfocus: data.onfocus, ...aria },
+        this._createUserIcon(),
+        this.renderLabel(data),
+        this.renderIcon(data)
+      );
+    }
```

Comment:
We could refactor it to enforce consistency:
```suggestion
    const children = [this._createUserIcon(), this.renderLabel(data)]
    if (dataset.empty !== 'true') {
      children.push(this.renderIcon(data));
    }
    
    return h.li(
      { className, dataset, tabindex: '0', onfocus: data.onfocus, ...aria },
      ...children
    );
```

> I remove the test `'empty' in dataset` as in that case `dataset.empty === undefined` -> we are covered